ut2004.desktop

```diff
@@ -1,8 +1,8 @@
 [Desktop Entry]
-Name=Unreal Tournament 2004
-Comment=popular first person shooter
+Name=UT2004
```

Comment:
Not sure I like this change as it is just an abbreviation of the actual name. I think we should leave it as Unreal Tournament 2004 personally.

In my case my fuzzy searching on my launcher finds Unreal Tournament 2004 if I type UT2004 regardless of this entry being Unreal Tournament 2004.

Reply:
So my reason for shortening it is that it gets pretty majorly cut off in Gnome 3 e.g. shows up as 'Unreal Tournament...' where UT2004 shows up exactly as it should. That being said I'm happy to revert because at least having the icon is an improvement :) but if you change your mind let me know.

Reply:
@jm2 this has now been pushed to the AUR. While I am working on this package I am going to look to see if there is a newer version of UT2004 though as I know https://www.oldunreal.com/ has started patching the original UT (not sure about UT2004 though but worth checking).

Reply:
With regards to the Gnome 3 truncating I think you could possibly tweak something to stop that. It doesn't affect KDE Plasma or the launcher I use in Sway (Rofi). I did some quick searching and found: https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/363 so might just be you need to update to Gnome 40 to resolve this.

Reply:
Unfortunately no new patches for UT2004 so we are still on the latest.

Reply:
Unfortunately I'm still seeing the truncating on the latest Gnome 40.x in Arch, but good to know it's been raised as a bug that may eventually get fixed.